commands/azureCommands/acr-logs-utils/tableDataManager.ts

```diff
@@ -50,7 +51,7 @@ export class LogData {
      * the next page of logs will be saved and all preexisting data will be deleted.
      * @param filter Specifies a filter for log items, if run Id is specified this will take precedence
      */
-    public async loadLogs(loadNext: boolean, removeOld?: boolean, filter?: Filter): Promise<void> {
+    public async loadLogs(webViewEvent: boolean, loadNext: boolean, removeOld?: boolean, filter?: Filter): Promise<void> {
```

Comment:
Not too fond of lots of bool args.  How about:

public async loadLogs(options: { isWebViewEvent: Boolean, loadNext: Boolean... etc.